settings/DevHome.Settings/ViewModels/ExtensionSettingsViewModel.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using AdaptiveCards.Rendering.WinUI3;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using CommunityToolkit.WinUI;
+using DevHome.Common.Extensions;
+using DevHome.Common.Services;
+using DevHome.Common.Views;
+using DevHome.Logging;
+using DevHome.Settings.Models;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.Windows.DevHome.SDK;
+using Windows.ApplicationModel;
+
+namespace DevHome.Settings.ViewModels;
+
+public partial class ExtensionSettingsViewModel : ObservableObject
+{
+    public ExtensionSettingsViewModel()
+    {
+        Breadcrumbs = new ObservableCollection<Breadcrumb> { };
+    }
+
+    public ObservableCollection<Breadcrumb> Breadcrumbs
+    {
+        get; set;
+    }
+
+    [RelayCommand]
+    private async Task OnSettingsContentLoadedAsync(PluginAdaptiveCardPanel pluginAdaptiveCardPanel)
+    {
+        var pluginWrappers = Task.Run(async () =>
+        {
+            var pluginService = Application.Current.GetService<IPluginService>();
+            return await pluginService.GetInstalledPluginsAsync(true);
+        }).Result;

```

Comment:
I had this, and Amir pointed out that you can use dependency injection to pass the PluginService into the constructor. 
Also, a question: why do we want to put this in a Task.Run()? I ended up just doing a regular await and it seemed fine.

Reply:
Oh nice, I didn't realize that!  Also, because I copy/pasted from somewhere else :)  Fixed.